src/accordion/accordion.ts

```diff
@@ -199,6 +207,14 @@ export class NgbAccordion implements AfterContentChecked {
    */
   isOpen(panelId: string): boolean { return this._states.get(panelId); }
 
+  /**
+   * @internal
```

Comment:
> In this case I left hidden because there are no examples of accordion component in Bootstrap 4 examples for Card and active would require writing custom CSS to hide the panel

I didn't find the example :P Anyway I'll make the proper changes then, thanks for pointing the example.